stream_data.py

```diff
@@ -0,0 +1,15 @@
+from gen_temperature_stream import *
+from gen_soc_stream import *
+
+def stream_data(number_of_streams):
+    for i in range(number_of_streams):
+        temperature_list = temperature_stream()
+        print(",".join([str(i) for i in temperature_list]))
+
+    for i in range(number_of_streams):
+        soc_list = soc_stream()
+        print(",".join([str(i) for i in soc_list]))
```

Comment:
Is it really required 2 for loops.  refer https://github.com/clean-code-craft-tcq-4/stream-line-Yashaswini1998/pull/1/files#r967862803